### PR TITLE
Update headless

### PR DESCRIPTION
**Pull request by magjed4289**

### CODE REVIEW

File: liferay/headless.py [hunk 1]
```diff
@@ -19,89 +18,99 @@ def _create_poshi_task_for(jira_local, parent_story, poshi_automation_table):
     print("Poshi task ", new_issue.key, " created for", parent_key)
 
 
-def create_creation_subtask(jira):
+def update_creation_subtask(jira):
     print("Creating test creation subtasks for Headless team...")
```
Comment (david-gutierrez-mesa) [inline liferay/headless.py:22]: Update instead of creating

File: liferay/headless.py [hunk 2]
```diff
@@ -19,89 +18,99 @@ def _create_poshi_task_for(jira_local, parent_story, poshi_automation_table):
...
-            child = jira.create_issue(fields=subtask_test_validation)
-            print("* Created sub-task: " + child.key)
-
-    print("Subtasks for Headless team are up to date")
+    stories_with_test_creation_subtask = jira.search_issues('filter=54996')
+    for story in stories_with_test_creation_subtask:
+        for subtask in story.fields.subtasks:
+            summary = subtask.fields.summary
+            key = subtask.key
+            test_creation_subtask_details = jira.search_issues("key =" + key)
```
Comment (david-gutierrez-mesa) [inline liferay/headless.py:28]: here you can simply do:
test_creation_subtask_details = jira.issue(key)

File: liferay/headless.py [hunk 3]
```diff
@@ -19,89 +18,99 @@ def _create_poshi_task_for(jira_local, parent_story, poshi_automation_table):
...
+            assignee = test_creation_subtask_details[0].fields.assignee
+            if summary == 'Test Scenarios Coverage | Test Creation':
+                print("Updating "+key+" ...")
+                if subtask.fields.status.name == "Open":
+                    description = '*Output*' \
+                                  '\r\n # Our table with the Test scenarios/test cases to be validated in the ' \
```
Comment (david-gutierrez-mesa) [inline liferay/headless.py:34]: It think it's better to add the \r\n at the end of the previous line so it's easier o read

File: liferay/headless.py [hunk 4]
```diff
@@ -19,89 +18,99 @@ def _create_poshi_task_for(jira_local, parent_story, poshi_automation_table):
...
+                                  '\r\n| | | | |' \
+                                  '\r\n' \
+                                  '\r\n*Exploratory testing to consider:*' \
+                                  '\r\n||Requirement||Test Scenarios||Test Priority (business impact)||Covered by ' \
+                                  'frontend/backend Unit Test?||' \
+                                  '\r\n| | | | |'
+                    subtask.update(fields={'description': description})
+                    if assignee != 'Support QA':
+                        jira.assign_issue(subtask.id, 'support-qa')
+                        break
```
Comment (david-gutierrez-mesa) [inline liferay/headless.py:54]: this is no needed. If we do enter here, next break is executed

File: liferay/headless.py [hunk 5]
```diff
@@ -19,89 +18,99 @@ def _create_poshi_task_for(jira_local, parent_story, poshi_automation_table):
...
+
+
+def update_validation_subtask(jira):
+    print("Updating test validation subtasks for Headless team...")
+    stories_with_test_validation_subtask = jira.search_issues('filter=55398')
+    for story in stories_with_test_validation_subtask:
+        for subtask in story.fields.subtasks:
+            summary = subtask.fields.summary
+            key = subtask.key
+            test_validation_subtask_details = jira.search_issues("key =" + key)
```
Comment (david-gutierrez-mesa) [inline liferay/headless.py:66]: here you can simply do:
test_creation_subtask_details = jira.issue(key)

Comment (david-gutierrez-mesa) [reply liferay/headless.py:66]: even better:
test_creation_subtask_details = jira.issue(key, fields='assignee')

File: liferay/headless.py (hunk 2 above)
Comment (david-gutierrez-mesa) [reply liferay/headless.py:28]: test_creation_subtask_details = jira.issue(key, fields='assignee')

File: liferay/headless.py [hunk 6]
```diff
@@ -19,89 +18,99 @@ def _create_poshi_task_for(jira_local, parent_story, poshi_automation_table):
...
+            assignee = test_validation_subtask_details[0].fields.assignee
+            if summary == 'Product QA | Test Validation - Round 1':
+                print("Updating "+key+" ...")
+                if subtask.fields.status.name == "Open":
+                    description = '\r\n*Context*' \
+                                  '\r\nExecute the tests of the parent story, and use the information in the *Test ' \
```
Comment (david-gutierrez-mesa) [inline liferay/headless.py:72]: same that before: It think it's better to add the \r\n at the end of the previous line so it's easier o read

File: liferay/headless.py [hunk 7]
```diff
@@ -19,89 +18,99 @@ def _create_poshi_task_for(jira_local, parent_story, poshi_automation_table):
...
+                                  '\r\n * *Impeditive:*' \
+                                  '\r\n||Ticket||Title||' \
+                                  '\r\n|?|?|' \
+                                  '\r\n * *Not Impeditive:*' \
+                                  '\r\n||Ticket||Title||' \
+                                  '\r\n|?|?|'
+                    subtask.update(fields={'description': description})
+                    if assignee != 'Support QA':
+                        jira.assign_issue(subtask.id, 'support-qa')
+                        break
```
Comment (david-gutierrez-mesa) [inline liferay/headless.py:111]: not needed break

**Review by david-gutierrez-mesa — CHANGES_REQUESTED** [review]
Good job! just some small changed required

File: liferay/headless.py (hunk 5 above)
Comment (magjed4289) [reply liferay/headless.py:66]: in this case I don't need the 
`assignee = test_validation_subtask_details[0].fields.assignee.displayName`
anymore? 🤔

Comment (david-gutierrez-mesa) [reply liferay/headless.py]: yes, but the you will only get assignee field

**Review by david-gutierrez-mesa — APPROVED** [review]

**Pull request merged**